### PR TITLE
Feature/6 dummy map

### PR DESCRIPTION
**Pull request by YaroslavAggressive**

### CODE REVIEW

File: backend/dummy_test_config.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[DEFAULT]
```
Comment (yarnaid) [inline backend/dummy_test_config.txt]: If it's test config, let's move it to the testing folder

File: backend/map_generation.py [hunk 2]
```diff
@@ -91,9 +93,9 @@ def create_generation_list(self) -> typing.List["HumanState"]:
             human_objects.append(HumanState.
                                  human_from_parameters(len(self.map_buildings) + len(self.map_population),
                                                        self.map_length, self.map_width))
-        return human_objects
+        self.map_population = human_objects
 
-    def has_intersection(self, new_building: Building) -> bool:
```
Comment (yarnaid) [inline backend/map_generation.py:96]: why did you remove type annotation?

File: backend/map_generation.py [hunk 3]
```diff
@@ -116,3 +118,25 @@ def get_population(self):
 
     def get_buildings(self):
         return self.map_buildings
+
+    def update_map(self):
+        pass
```
Comment (yarnaid) [inline backend/map_generation.py]: do we need this function for now? Maybe remove it or `raise NotImplementedError`?

File: backend/map_generation.py [hunk 4]
```diff
@@ -116,3 +118,25 @@ def get_population(self):
...
+
+
+def create_dummy_map() -> "ResearchMap":
```
Comment (yarnaid) [inline backend/map_generation.py]: We can annotate return type without quotes

File: backend/dummy_test_config.txt (hunk 1 above)
Comment (YaroslavAggressive) [reply backend/dummy_test_config.txt]: Done

File: backend/map_generation.py (hunk 2 above)
Comment (YaroslavAggressive) [reply backend/map_generation.py:96]: Done

File: backend/map_generation.py (hunk 3 above)
Comment (YaroslavAggressive) [reply backend/map_generation.py]: Done (removed)

File: backend/map_generation.py (hunk 4 above)
Comment (YaroslavAggressive) [reply backend/map_generation.py]: Done
